snprintf is C99

--- origsrc/pygobject-3.14.0/gi/pygi-basictype.c	2014-09-10 17:10:32.000000000 -0500
+++ src/pygobject-3.14.0/gi/pygi-basictype.c	2015-01-04 03:00:27.511771100 -0600
@@ -81,7 +81,7 @@ check_valid_double (double x, double min
             PyErr_Clear ();
 
         /* we need this as PyErr_Format() does not support float types */
-        snprintf (buf, sizeof (buf), "%g not in range %g to %g", x, min, max);
+        g_snprintf (buf, sizeof (buf), "%g not in range %g to %g", x, min, max);
         PyErr_SetString (PyExc_OverflowError, buf);
         return FALSE;
     }
